src/tools/r2rdump/XmlDumper.cs

```diff
@@ -25,6 +28,23 @@ public XmlDumper(R2RReader r2r, TextWriter writer, bool raw, bool header, bool d
             _unwind = unwind;
             _gc = gc;
             _sectionContents = sectionContents;
+
+            _ignoredProperties = new XmlAttributeOverrides();
```

Comment:
Currently this will always ignore those properties. Did you mean
`attrs.XmlIgnore = ignoreSensitive;` ?

Reply:
oops, I'll fix that